R/pkg/R/generics.R

```diff
@@ -942,6 +942,14 @@ setGeneric("countDistinct", function(x, ...) { standardGeneric("countDistinct")
 #' @export
 setGeneric("crc32", function(x) { standardGeneric("crc32") })
 
+#' @rdname create_array
```

Comment:
It covers all `o.a.s.sql.functions` right now. I am not sure these two are different enough to be an exception (and what about `struct` which belongs to the same category).